Makefile

```diff
@@ -34,11 +34,10 @@ help:
 	@echo "Internationalization"
 	@echo "--------------------"
 	@echo ""
-	@echo "translation-django-makemessages: creates source messages for django"
-	@echo "translation-django-compilemessages: compiles all language translation sources"
```

Comment:
But this command is necessary to run separately from the downloading from CrowdIn. We have the `.po` files kept in the repo, but to generate the `.mo` files (in the `dist` target amongst others), we need to be able to run this from a platform that doesn't fetch anything from CrowdIn (requiring API access and crowdin-cli).

Reply:
okay sounds good